docs/learn/learn-xcm.md

```diff
@@ -0,0 +1,299 @@
+---
+id: learn-xcm
+title: Cross-Consensus Message Format (XCM)
+sidebar_label: Cross-Consensus Message Format (XCM)
+description: Learn about the messaging format at the forefront of interoperability.
+keywords: [cross-consensus, XCM, XCMP, interoperability, communication]
+slug: ../learn-xcm
+---
+
+Cross-Consensus Message Format(XCM) aims to be a language to communicate ideas between consensus systems. One
+of Polkadot's promises is that of interoperability, and XCM is the vehicle by which it will deliver this
+promise. Simply, it is a standard that allows protocol developers to define the data and origins which
+their chains can send and recieve from. Out of the box it comes with a VM that allows for customization
+of execution as well as the following properties:
+
+1. **Asynchronous**: XCM messages in no way assume that the sender will be blocking on its
+   completion.
+2. **Absolute**: XCM messages are guaranteed to be delivered and interpreted accurately, in order
+   and in a timely fashion.
+3. **Asymmetric**: XCM messages do not have results. Any results must be separately communicated to
```

Comment:
"XCM messages do not have results." - Does this mean XCM messages are not acknowledged by the receiver?

The statement as is, is ambiguous and imply that they are inconsequential

Reply:
Correct - from what I understand as a message, it can be sent but out of the box there is not a callback function that lets the origin know that it was received. Though this can be programmed through the XCVM if needed.

Reply:
I've added a bit more info to explain this.